public/app/features/alerting/unified/components/receivers/form/GrafanaReceiverForm.tsx

```diff
@@ -160,6 +165,9 @@ export const GrafanaReceiverForm = ({ contactPoint, readOnly = false, editMode }
         defaultItem={{ ...defaultChannelValues }}
         commonSettingsComponent={GrafanaCommonChannelSettings}
         customValidators={{ [ReceiverTypes.OnCall]: onCallFormValidators }}
+        canManagePermissions={
+          editMode && contactPoint && showManageContactPointPermissions(GRAFANA_RULES_SOURCE_NAME, contactPoint)
```

Comment:
why we don't allow editing permissions while creating a new contact point? is this intentional?

Reply:
We can't modify the permissions before creation as we don't have an ID to use for the access-control APIs